### PR TITLE
Update Auth client-api.md

### PR DESCRIPTION
**Pull request by llbartekll**
As discussed on yesterday's sdk meeting.

### CODE REVIEW

**Review by jakobuid — APPROVED** [review]

**Review by jakobuid — APPROVED** [review]

File: auth/client-api.md [hunk 1]
```diff
@@ -18,9 +18,12 @@ abstract class Client {
...
   public abstract getPendingRequests(): Promise<Record<number, PendingRequest>>;
-
-  // query cached response matching id
-  public abstract getResponse(params: { id: number }): Promise<Response>;
+  
+  // querry pairings
+  public abstract getPairings(): Promise<Array<Pairing>>;
+  
+  // for either to ping a peer in a pairing
+  public abstract ping(params: { topic: string; }): Promise<void>;
```
Comment (jakubuid) [inline auth/client-api.md:26]: If we let users to get list of pairings should we also allow them to delete one? Imo It's be cool

Comment (llbartekll) [reply auth/client-api.md:26]: make sense to me, I will add it!

File: auth/client-api.md [hunk 2]
```diff
@@ -18,9 +18,16 @@ abstract class Client {
...
   // query all pending requests
   public abstract getPendingRequests(): Promise<Record<number, PendingRequest>>;
+  
+  // querry pairings
+  public abstract getPairings(): Promise<Array<Pairing>>;
+  
+  // for either to ping a peer in a pairing
+  public abstract ping(params: { topic: string; }): Promise<void>;
+  
+  // for either to disconnect a pairing
```
Comment (bkrem) [inline auth/client-api.md]:
```suggestion
  // for either peer to disconnect a pairing
```

File: auth/client-api.md [hunk 3]
```diff
@@ -18,9 +18,16 @@ abstract class Client {
 
   // query all pending requests
   public abstract getPendingRequests(): Promise<Record<number, PendingRequest>>;
+  
+  // querry pairings
```
Comment (bkrem) [inline auth/client-api.md]:
```suggestion
  // query pairings
```

File: auth/client-api.md [hunk 4]
```diff
@@ -18,9 +18,16 @@ abstract class Client {
...
+  public abstract getPairings(): Promise<Array<Pairing>>;
+  
+  // for either to ping a peer in a pairing
+  public abstract ping(params: { topic: string; }): Promise<void>;
+  
+  // for either to disconnect a pairing
+  public abstract disconnect(params: { topic: string, reason: Reason}): Promise<void>;
```
Comment (bkrem) [inline auth/client-api.md]: Should we add `Reason` to the data structures?

**Review by bkrem — APPROVED** [review]

File: auth/client-api.md (hunk 4 above)
Comment (llbartekll) [reply auth/client-api.md]: actually, the reason will be always .userDisconnected 
so what about:
```suggestion
  public abstract disconnect(params: { topic: string }): Promise<void>;
```

Comment (llbartekll) [reply auth/client-api.md]: @bkrem ⬆️

Comment (bkrem) [reply auth/client-api.md]: True since it’s an explicit top level action here it’s always user initiated 👍

**Pull request merged**